core/src/banking_stage/unprocessed_packet_batches.rs

```diff
@@ -89,26 +84,6 @@ impl UnprocessedPacketBatches {
         self.message_hash_to_transaction.clear();
     }
 
-    /// Insert new `deserialized_packet_batch` into inner `MinMaxHeap<DeserializedPacket>`,
```

Comment:
this type `UnprocessedPacketBatches` does not need to exist anymore

Reply:
`DeserializedPacket` is also useless 😄 but we can leave that for a follow-up PR because it involves a bunch of small changes.

Reply:
^it is useless because it wraps a `Arc<ImmutableDeserializedPacket>` with a `forwarded` bool; which is no longer part of banking stage's responsibilities